TransIT.BLL/ConfigureBLLExtension.cs

```diff
@@ -100,5 +104,18 @@ private static void ConfigureAutoMapper(this IServiceCollection services)
                 c.AddProfile(new LocationProfile());
             }).CreateMapper());
         }
+
+        private static void ConfigureDocumentService(
+            this IServiceCollection services,
+            IConfiguration configuration)
+        {
+            services.Configure<DocumentServiceOptions>((options) => {
+                var documentServiceOptions = configuration.GetSection(nameof(DocumentServiceOptions));
+
+                long maxmimumSize = Convert.ToInt64(documentServiceOptions[nameof(options.MaximumSize)]);
```

Comment:
please use save parsing approach, because convert class throws exception in case of error parsing

use like this
```

  options.MaximumSize = long.TryParse(documentServiceOptions[nameof(options.MaximumSize)], out var size) && size > 0 
                                      ? size
                                      : long.MaxValue;
```

Reply:
Convert class is used for reason, person that configures application should know if specified value is incorrect

Reply:
the value configures not user but developed or devops, so why user should see the error?